source/speechViewer.py

```diff
@@ -97,10 +97,8 @@ def _onDialogActivated(self, evt):
 			self.shouldShowOnStartupCheckBox.SetFocus()
 
 	def onClose(self, evt):
-		if not evt.CanVeto():
-			deactivate()
-			return
-		evt.Veto()
+		assert isActive

```

Comment:
```suggestion
		assert isActive, "Cannot close Speech Viewer as it is already inactive"
```

It would be helpful to have a message here, but deactivate should still work (as a no-op) regardless of the value `isActive` right?

Reply:
@seanbudd wrote:
> deactivate should still work (as a no-op) regardless of the value of isActive right?

I confirm it is the case.